js/scenes/FTU.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+  Image,
+  Text,
+  Button,
+  Dimensions,
+  StyleSheet,
+  Platform,
+} from 'react-native';
+import ViewPager from 'react-native-viewpager';
+import ViewPagerIndicator from '../components/ViewPagerIndicator';
+import { connect } from 'react-redux';
+import { theme } from '../../config';
+
+export class FTU extends Component {
+  constructor(props) {
+    super(props);
+
+    this.navigator = this.props.navigator;
+
+    const PAGES = [
```

Comment:
The pages are rendered on demand. The code used to build each page is only called when required (think of `<ListView>` component).
The API seems flexible so it can also handle complicated case where you go to different pages depending on which button you tap.